app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        person_instance = Person.people[person["name"]]
```

Comment:
Accessing `Person.people[person['name']]` assumes that the name key exists in the dictionary. If a name is not found, this will raise a KeyError. Consider adding error handling to manage such cases.